static/src/stylesheets/module/reader-revenue/_moment-epic.scss

```diff
@@ -0,0 +1,8 @@
+// TBD: Add all moment epic class names here when they are available
+[class^='contributions__epic--2019-10_contribs_environment-pledge-moment'],
+.contributions__epic--2019-10_contribs_environment-pledge-moment_global_epic_hook {
```

Comment:
I just had to googe the `^=` operator, but am I right in thinking line 3 here is redundant, because line 2 matches all classes beginning with `contributions__epic--2019-10_contribs_environment-pledge-moment`?

Reply:
yes it's a strange regex-lite thing you can do in CSS, and yes, you're right it's redundant - will clean that up